src/outputs/short_form_out.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+import numpy as np
+
+
+def create_new_cols(df: pd.DataFrame) -> pd.DataFrame:
+    """Created blank columns & year column for short form output
+
+    Args:
+        df (pd.DataFrame): short form output data frame
+
+    Returns:
+        pd.DataFrame: returns short form output data frame with added new columns
+    """
+    columns_names = [
```

Comment:
ticket mentions using a dict but is this preferable? Was there an idea behind using a dict?